docs/getting-started-guides/docker.md

```diff
@@ -0,0 +1,71 @@
+## Running kubernetes locally via Docker
```

Comment:
I did test it on my mac, and it does work correctly!  You need to boot2docker ssh -L8080:localhost:8080 to access it from outside of the VirtualBox VM, I updated instructions.